python/lsst/ts/m2gui/controltab/tab_actuator_control.py

```diff
@@ -33,7 +57,516 @@ class TabActuatorControl(TabDefault):
         Table's title.
     model : `Model`
         Model class.
+
+    Attributes
+    ----------
+    model : `Model`
+        Model class.
     """
 
+    MAX_DISPLACEMENT = 10**7
+
     def __init__(self, title, model):
         super().__init__(title, model)
+
+        self._info_script = {
+            "file": create_label(""),
+            "progress": self._create_progress_bar(0, 100, 0),
+        }
```

Comment:
Those are default (0, 100, 0). You can spare function and ~10 lines by replacing that with QProgressBar()

Reply:
The default value of `QProgressBar()` is -1 instead of 0.

Reply:
Update in the commit: [854d782](https://github.com/lsst-ts/ts_m2gui/pull/12/commits/854d7823b36c3edc5c9f8408fe5d0011a77139ea). Thanks!